### PR TITLE
Update the README.md file to describe the repo.

### PR DESCRIPTION
**Pull request by JabariD**
First pull request in STEP portfolio repo. Personalized README.md
@ptai7

### CODE REVIEW

**Comment by JabariD** [comment]
PR's should never be this long. Sorry about this! @chrisc94

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,3 +1,11 @@
+html {
+    background-color: gainsboro;
+}
+
+body {
+    font-family: 'Courier New', Courier, monospace;
+}
+
 #content {
   margin-left: auto;
```
Comment (chrisc94) [inline portfolio/src/main/webapp/style.css:10]: nit: the tabs here and above look slightly inconsistent. could we decide on either 2 spaces or 4?

Comment (JabariD) [reply portfolio/src/main/webapp/style.css]: Gotcha!

**Comment by JabariD** [comment]
This PR is **incredibly long**, sorry about that. Still getting used to how GitHub works in terms of branching, PR's, and commits. @ptai7 @chrisc94 

But I'll be starting at **_Week 3: Server_ : Part 4 - HTML Forms and Post Requests** Monday in a new branch.

Everything from Week 1 - Week 3: Part 3 has been completed and is in this huge PR. And I'll definitely study how branches and commits work.

**Pull request merged**